js_modules/dagster-ui/packages/ui-components/src/components/NonIdealState.tsx

```diff
@@ -31,6 +31,7 @@ export const NonIdealState = ({
       flex={{gap: 20, alignItems: singleContentElement ? 'center' : 'flex-start'}}
       background={Colors.backgroundLight()}
       padding={24}
+      className="non-ideal-state-parent-box"
```

Comment:
Actually, the styled component way to do this would be to make a 
```
export const NonIdeaStateWrapper = styled(Box)``;
```

and then in the other component use can do `
``
${NonIdeaStateWrapper} {
// ...custom css here
}
```

Reply:
Thanks, hope my recent commit is what you wanted